unity/Assets/Scripts/AgentManager.cs

```diff
@@ -952,6 +954,7 @@ public class ThirdPartyCameraMetadata
 	public int thirdPartyCameraId;
 	public Vector3 position;
 	public Vector3 rotation;
+	public float fieldOfView;
```

Comment:
this should be nullable, but when i tried, the serializer stopped emitting it.

Reply:
to python? I believe it works for `gridColor` but I am not sure.

If you already tried this then it's ok to use zero. You could also wrap it in another class `class Float` that uses the `[Serializable]` directive to box it and allow for null values.

Reply:
gridColor is never read in python... my guess is that this is not working as expected... but i could be wrong
maybe ill dig in and make a test to see